include/Foundation/NSURLSession.h

```diff
@@ -181,27 +181,27 @@ FOUNDATION_EXPORT_CLASS
 @protocol NSURLSessionDataDelegate <NSURLSessionTaskDelegate>
 @optional
 - (void)URLSession:(NSURLSession*)session
-          dataTask:(NSURLSessionDataTask*)dataTask
-didReceiveResponse:(NSURLResponse*)response
- completionHandler:(void (^)(NSURLSessionResponseDisposition disposition))completionHandler;
+              dataTask:(NSURLSessionDataTask*)dataTask
+    didReceiveResponse:(NSURLResponse*)response
+     completionHandler:(void (^)(NSURLSessionResponseDisposition disposition))completionHandler;
 - (void)URLSession:(NSURLSession*)session
                  dataTask:(NSURLSessionDataTask*)dataTask
     didBecomeDownloadTask:(NSURLSessionDownloadTask*)downloadTask;
 - (void)URLSession:(NSURLSession*)session dataTask:(NSURLSessionDataTask*)dataTask didReceiveData:(NSData*)data;
 - (void)URLSession:(NSURLSession*)session
-          dataTask:(NSURLSessionDataTask*)dataTask
- willCacheResponse:(NSCachedURLResponse*)proposedResponse
- completionHandler:(void (^)(NSCachedURLResponse* cachedResponse))completionHandler;
+             dataTask:(NSURLSessionDataTask*)dataTask
+    willCacheResponse:(NSCachedURLResponse*)proposedResponse
+    completionHandler:(void (^)(NSCachedURLResponse* cachedResponse))completionHandler;
 @end
 
 @protocol NSURLSessionDownloadDelegate <NSURLSessionTaskDelegate>
 - (void)URLSession:(NSURLSession*)session downloadTask:(NSURLSessionDownloadTask*)downloadTask didFinishDownloadingToURL:(NSURL*)location;
 
 @optional
 - (void)URLSession:(NSURLSession*)session
-      downloadTask:(NSURLSessionDownloadTask*)downloadTask
- didResumeAtOffset:(int64_t)fileOffset
-expectedTotalBytes:(int64_t)expectedTotalBytes;
+          downloadTask:(NSURLSessionDownloadTask*)downloadTask
+     didResumeAtOffset:(int64_t)fileOffset
+    expectedTotalBytes:(int64_t)expectedTotalBytes;
```

Comment:
this formatting is objectively worse than the one that existed before :(

objective-c often lines up on colons; I noticed that clang-format wasn't doing that in a prior PR, but didn't think much of it.